.doc_gen/metadata/sesv2_metadata.yaml

```diff
@@ -0,0 +1,97 @@
+# zexi 0.4.0
+sesv2_CreateContactList:
+  title: Create an &SESv2; contact list using an &AWS; SDK
+  title_abbrev: Create a contact list
+  synopsis: create an &SESv2; contact list.
+  category:
+  languages:
+    Rust:
+      versions:
+        - sdk_version: 1
+          github: .rust_alpha/ses
```

Comment:
The code examples list in your [README](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/.rust_alpha/ses#code-examples) isn't rendering right.

Reply:
Looks good to me. I'll ask someone else to also take a look.